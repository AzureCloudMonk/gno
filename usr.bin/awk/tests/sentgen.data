Sentence
Sentence
Nounphrase
Verb
Adverb
BadElement
Sentence
