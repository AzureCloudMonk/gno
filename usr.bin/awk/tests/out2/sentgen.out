the boy runs quickly 
the boy walks very very very quickly 
the boy 
walks 
quickly 
unknown nonterminal: BadElement
the girl runs slowly 
